test/test_gpio.c

```diff
@@ -4,35 +4,41 @@
 
 #include "unity.h"
 
-void setup_test(void) { setbuf(stdout, NULL); }
+void setup_test(void) {}
 
 void teardown_test(void) {}
 
 // gpio_init
 
 // Tests that the GPIO init works. This should simply return true 100% of time unless the test is on
 // x86 and the configuration is incorrect in which case fix it!
-void test_gpio_init(void) { TEST_ASSERT_TRUE(gpio_init()); }
+void test_gpio_valid_init(void) { TEST_ASSERT_TRUE(gpio_init()); }
```

Comment:
Yeah, sounds like something we should add. In general, my tests are more like test\_[module]\_[purpose], where in this case purpose would be [function]_[test].